### ISSUE TITLE
Upgrade to Guzzle 6

### ISSUE THREAD

**Issue by elazar** [marketo-api/marketo-rest-client#19]
Guzzle 3 hasn't seen a release in over 2 years. PHP 5.3 has been EOL for nearly 3 years.

**Comment by Jaesin** [comment]
Agreed. Although I've been using [Guzzle 6 services](https://github.com/guzzle/guzzle-services) for a while now. It would be very simple if the service definition file was improved to reduce the custom code in this client.

No one is currently working on the port. PRs are welcome.
